tests/e2e/test_api.py

```diff
@@ -728,6 +728,29 @@ def test_forbidden_user():
     assert response.status_code == requests.codes.forbidden
 
 
+# TODO: disabled until we figure out how to get the token that returns
+# user kube:admin in the CI
```

Comment:
we can at least have a legitimate test of the get_cluster_id helper, right?  (not really an e2e test since you won't be going through the OLS server, but since we have a cluster stood up, you can test that the get_cluster_id helper gets the right value.